public/js/test/stubNetworkRequest.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
+
+module.exports = function(url) {
+  return new Promise((resolve, reject) => {
+    // example.com is used at a dummy URL that points to our local
+    // `/public/js` folder.
+    url = url.replace("http://example.com/test/", "/../js/test/mochitest/examples/");
+    resolve({ content: fs.readFileSync(__dirname + url, "utf8") });
+  });
+};
```

Comment:
sorry, if this was not clear, i was suggesting moving this module to the dir because it's a utility